docs/getting-started/enterprise/splunk-app.md

```diff
@@ -0,0 +1,104 @@
+# Splunk App
+
+The Bitwarden Splunk app fetches event log data from the Bitwarden Public API and makes it available
+in Splunk.
+
+## Requirements
+
+- Docker. If you're using an Apple Silicon Mac, enable _Docker Desktop_ -> _Settings_ -> _General_
+  -> _Use Rosetta for x86_64/amd64 emulation on Apple Silicon_
+- Python 3.8 or 3.9
+- [Poetry][poetry]
+- libmagic (macOS only), available via homebrew: `brew install libmagic`
+- A Bitwarden server with event logging configured and an enterprise organization. To set this up
```

Comment:
- Event logging works out of the box, there is nothing to configure - unless it's some custom self-hosted setup, which does not have event logging enabled.
- Event logging works on any business plan, including Teams - makes me wonder if this does belongs to the "Enterprise" section 🤔

Reply:
* event logging - that's true, I've amended this to be more specific. Many devs, myself included, just run the minimum api & identity projects when developing day-to-day, so I want to highlight the additional requirements here
* plan requirements - also true! I've fixed this. The "Enterprise" category is meant to describe enterprise use (business use), not the enterprise plan specifically, maybe it could be better named. Directory Connector is under this category and that is also compatible with Teams plans.